packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -83,10 +71,7 @@ const config = {
 
 describe('Auth', () => {
     beforeEach(() => {
-        mockRefreshAccessToken.mockClear()
-        mockLoginGuestUser.mockClear()
-        mockLoginRegisteredUserB2C.mockClear()
-        mockLogout.mockClear()
+        jest.clearAllMocks()
```

Comment:
This isn't handled here?

https://github.com/SalesforceCommerceCloud/pwa-kit/pull/825/files#diff-8dc2e0805fd1db070dc61186e9035af85751d5a2d715ea80d0a1e543c36f70ffL43

Reply:
Nope, the link you highlighted cleans nock mocks, but here we are cleaning the Jest module mock for the `Auth` class.